Functions/xASL_adm_GetDeprecatedFields.m

```diff
@@ -85,11 +86,14 @@
     'bSkipIfNoM0'                   'settings'  'bSkipIfNoM0'                        '';...
     'bSkipIfNoFlair'                'settings'  'bSkipIfNoFlair'                     '';...
     'bRunModule_DARTEL'             'modules'   'bRunDARTEL'                         '';...
+    'bRunDARTEL'                    'modules'   'bRunDARTEL'                         '';...
```

Comment:
That's modules, not modules.structural? Just checking.

Reply:
I added that line to make sure that all the fields are moved correctly within `Functions/xASL_bids_parms2BIDS`. So If we move `x.modules.bRunDARTEL` to the base level of x e.g., then we can't move bRunModule_DARTEL back. I could have done a better job describing this though.

Reply:
Totally fine. You can resolve. I was just double-checking...